robot_A: absent
robot_B: attend
robot_C: absent
robot_D: attend
robot_E: absent